app/assets/stylesheets/modules/form.scss

```diff
@@ -0,0 +1,20 @@
+@import "variables";
+@import "mixins";
+
+.sign-up {
+  width: 50%;
+  margin-left: auto;
+  margin-right: auto;
+
+  .legend {
+    font-family: $dogstay-font-theme;
+    font-size: 2em;
+  }
+
+  .postfix {
+    background: #008080;
```

Comment:
Color literals like `#008080` should only be used in variable declarations; they should be referred to via variable everywhere else.